lua/conform/formatters/easy-coding-standard.lua

```diff
@@ -0,0 +1,18 @@
+local util = require("conform.util")
+
+---@type conform.FileFormatterConfig
+return {
+  meta = {
+    url = "https://github.com/easy-coding-standard/easy-coding-standard",
+    description = "ecs - Use Coding Standard with 0-knowledge of PHP-CS-Fixer and PHP_CodeSniffer.",
+  },
+  command = util.find_executable({
+    "vendor/bin/ecs",
+  }, "easy-coding-standard"),
```

Comment:
Is the executable `ecs`? If so, the last argument to find_executable should be `ecs` because that's the fallback if none of the listed names are found

Reply:
Yes, it is. Resolved.